atat_internal_api.yaml

```
@@ -0,0 +1,2197 @@
+          enum:
+            - DRAFT
+            - ACTIVE
+            - ARCHIVED
+    PortfolioDetail:
+      allOf:
+        - $ref: "#/components/schemas/Portfolio"
+        - $ref: '#/components/schemas/PortfolioSummary'
+        - $ref: '#/components/schemas/PortfolioAccess'
+    Application:
+      required:
+        - environments
+        - name
+        - operators
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/BaseObject"
+      properties:
+        environments:
+          minItems: 1
+          type: array
+          items:
+            $ref: "#/components/schemas/Environment"
+        name:
+          pattern: "^[a-zA-Z\\d _-]{4,100}$"
+          type: string
+        description:
+          pattern: "^[\\w\\d !@#$%^&*_|:;,'.-]{0,300}$"
+          type: string
+      additionalProperties: false
+      description: "Represents an Application in a Portfolio"
+    ApplicationDetail:
+      allOf:
+        - $ref: '#/components/schemas/Application'
+        - $ref: '#/components/schemas/AppEnvAccess'
+    Environment:
+      required:
+        - name
+        - operators
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/BaseObject"
+      properties:
+        name:
+          pattern: "^[a-zA-Z\\d _-]{1,100}$"
+          type: string
+      additionalProperties: false
+      description: "Represents an Environment for a specific Application"
+    EnvironmentDetail:
+      allOf:
+        - $ref: '#/components/schemas/Environment'
+        - $ref: '#/components/schemas/AppEnvAccess'
+    Clin:
+      required:
+        - clin_number
+        - idiq_clin
+        - obligatedFunds
+        - popEndDate
+        - popStartDate
+        - totalClinValue
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/BaseObject"
+      properties:
+        clinNumber:
+          pattern: "(?!^0{4}$)^\\d{4}$"
+          type: string
+          description: "Contract Line Item Number (CLIN), 0001 through 9999"
+        obligatedFunds:
+          type: number
+        idiqClin:
+          type: string
+        popStartDate:
+          type: string
+          description: "Start of period of performance"
+          format: date
+        popEndDate:
+          type: string
+          description: "End of period of performance"
+          format: date
+        totalClinValue:
+          type: number
+      additionalProperties: false
+      description: "Represents a Clin in a Task Order"
+    TaskOrder:
+      required:
+        - clins
+        - taskOrderFile
+        - taskOrderNumber
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/BaseObject"
+      properties:
+        clins:
+          minItems: 1
+          type: array
+          items:
+            $ref: "#/components/schemas/Clin"
+        taskOrderFile:
+          description: "Metadata associated with file which was previously uploaded\
+            \ by POST /taskOrderFiles"
```

```suggestion
          description: >-
            Metadata associated with file which was previously uploaded
            by POST /taskOrderFiles
```

https://yaml-multiline.info/

There are a few other places throughout this file where we should do this too. A search for `\s+\\` should turn most of them up.